src/gui/autoexpandabledialog.cpp

```diff
@@ -57,6 +57,13 @@ QString AutoExpandableDialog::getText(QWidget *parent, const QString &title, con
     d.m_ui->textEdit->setEchoMode(mode);
     d.m_ui->textEdit->setInputMethodHints(inputMethodHints);
 
+    const int lastDotIndex = text.lastIndexOf('.');
+    // select file name without extension, except 'extension'-only files like .gitignore
```

Comment:
.gitignore is not extension-only file, it is a hidden file (sort of) on *nix systems.
https://askubuntu.com/questions/94780/what-are-dot-files

Maybe: select file name without extension, except dot files like .gitignore

Reply:
Didn't even read that comment, just copy pasted :) will change.